/**:
  ros__parameters:
    input_topic: /velodyne_points
    output_topic: /velodyne_points_fixed
